build/custom/phing/build.xml

```diff
@@ -3,7 +3,7 @@
 <project name="blt" default="list">
 
   <!-- Define blt root directory. -->
-  <property name="blt.root.relative" value="${project.basedir}/../.."/>
+  <property name="blt.root.relative" value="${project.basedir}/../../.."/>
```

Comment:
Why has this been moved into a custom dir? I don't anticipate a case where someone would add custom targets here. If we must use the core/custom paradigm here, this file would belong in build/core.